agent/consul/state/catalog.go

```diff
@@ -1366,6 +1533,12 @@ func (s *Store) NodeCheck(nodeName string, checkID types.CheckID) (uint64, *stru
 	tx := s.db.Txn(false)
 	defer tx.Abort()
 
+	return s.nodeCheckTxn(tx, nodeName, checkID)
+}
+
+// nodeCheckTxn is used as the inner method to handle reading a health check
+// from the state store.
+func (s *Store) nodeCheckTxn(tx *memdb.Txn, nodeName string, checkID types.CheckID) (uint64, *structs.HealthCheck, error) {
```

Comment:
Same thing as nodeServiceTxn. Should this be getNodeCheckTxn to be consistent with getNodeTxn. Or maybe getNodeTxn should just be nodeTxn.